projects/xtoken/index.js

```diff
@@ -96,6 +98,25 @@ async function tvl(timestamp, block) {
     .formatUnits(combinedXaaveAave, 0)
     .split(".")[0];
 
+  // xALPHa created at block 13401087
+  let xalphaaNav = ethers.BigNumber.from("0");
+  if (block >= 13401087) {
```

Comment:
nitpick: we could have created a self explaining variable and skipped the comment.
```js
const XALPHA_CREATION_BLOCK = 13401087
```

Reply:
For sure I'll make this change. There are a bunch of special blocks previously hardcoded, but no reason not to name them from now on.